deploy-web/src/utils/apiUtils.ts

```diff
@@ -58,10 +58,10 @@ export class ApiUrlService {
     return `${apiEndpoint}/cosmos/feegrant/v1beta1/allowances/${address}`;
   }
   static dashboardData() {
-    return `${BASE_API_URL}/dashboardData`;
+    return `${BASE_API_URL}/dashboard-data`;
```

Comment:
Should we version apiUtils as well? I see that you added the version prefix in the BASE_API_URL variable but it applies to all the routes. If we add or change routes between versions, it might break these routes or I guess we can change the apiUtils to the new versions of the routes.

Reply:
We should not end up calling different versions of the api in the deploy tool. If we publish a new version of the api, we would update `apiUtils.ts` to use the latest version everywhere.